xlators/mgmt/glusterd/src/glusterd-gfproxyd-svc.c

```diff
@@ -327,6 +327,10 @@ glusterd_gfproxydsvc_start(glusterd_svc_t *svc, int flags)
         if (strcmp(localtime_logging, "enable") == 0)
             runner_add_arg(&runner, "--localtime-logging");
     }
+    if (this->ctx->log.logger == gf_logger_syslog) {
```

Comment:
```suggestion
    if (global_ctx->log.logger == gf_logger_syslog) {
```

Please, use the `global_ctx` instead of getting the ctx from the xlator. It's faster and removes the need for `THIS` in some places.

Reply:
Ack.